README.md

```diff
@@ -327,6 +327,49 @@ interface. In case of a missing `interface_name`, the `name` of the profile name
 **Note:** The `name` (the profile name) and the `interface_name` (the device name) may be
 different or the profile may not be tied to an interface at all.
 
+### `match`
+
+Settings to specify devices or systems matching a profile. Currently, only the `path`
+setting is implemented.
+
+The settings support a list of patterns which support the following modifiers and
+wildcards:
+
+**Special modifiers for `match` settings:**
+
+- `|`, the element is optional, the match evaluates to be true if at least one of the
+  optional element matches (logical OR). By default, an element is optional. This means
+  that an element `foo` behaves the same as `|foo`
```

Comment:
Defining the elements as optional is irritating, since at least one needs to match. So they are alternatives. Optional would mean that each element can match but none is required, in my opinion.

Reply:
Updated.